### ISSUE TITLE
No example

### ISSUE THREAD

**Issue by faribauc** [simplifiedcourses/template-driven-forms-starter#1]
Hi!

It would be really nice if this repo had a concrete working implementaion, instead of seeing "Enter your stuff here" in the main template file.

The 3 articles on your site have naming discrepencies between them in the examples and it's hard to follow what gets removed from one the next.

For example, the article "Say goodbye to custom form validators in Angular" (https://blog.simplified.courses/say-goodbye-to-custom-form-validators-in-angular/) uses a `model` directive, but the name in the code is actually `ngModel`. There is also no reference in the code to what the `vm.form` or `suite` variables are suppised to be. I would assume the former refers to `viewModel.user` from the previous "Template-driven or reactive forms in Angular" (https://blog.simplified.courses/template-driven-or-reactive-forms-in-angular/) article, which defines a `user` property inside `viewModel`, and that the later refers to `userValidations` defined earlier in the article, but consistency with naming things would be really helpful in following the code changes throughout this series.

I tried the simpletest implementation but it's throwing an error:

https://stackblitz-starters-cq3vcg.stackblitz.io

I do get the same error when I put this simple form into the `app.component.html` template file and run the project with the original `package.json`.

I don't mean to be negative, I really like this method of replacing the whole Angular FormControl thing with signals, but I was hoping to see working code samples in here.

Anyway, just wanted to point out a few things. If anything can be done, it would be awesome!

Thanks!

**Comment by simplifiedcourses** [comment]
Hi There,

So, the goal of this repo is to be as clean as possible, a starter template without any actually working code.
If you want a working solution I have this one for you: https://github.com/simplifiedcourses/template-driven-forms
I also have a Stackblitz as well https://stackblitz.com/~/github.com/simplifiedcourses/template-driven-forms
This should contain real use cases with real working code.
I get your frustration, because while writing this Your link keeps "loading"😅

Look, if my links don't help you. Send me an email at my personal email billietbrecht@gmail.com, and I'll gladly help out!

Cheers!
Brecht

**Comment by faribauc** [comment]
Hi Bretch,

Thanks for pointing me to that other repo, there is indeed a lot more meat in there. I'll give it a go for sure!

Cheers!
